assets/js/components/Dashboard/NewPaMessage/RouteColumn.tsx

```diff
@@ -0,0 +1,79 @@
+import React from "react";
+import fp from "lodash/fp";
+import type { Place } from "Models/place";
+import { sortByStationOrder } from "../../../util";
+import { Form } from "react-bootstrap";
+
+const RouteColumn = ({
+  label,
+  routes,
+  orderingRoute = routes[0],
+  places,
+  value,
+  onChange,
+  reverse = false,
+}: {
+  label: string;
+  routes: string[];
+  orderingRoute?: string;
+  places: Place[];
+  value: string[];
+  onChange: (zones: string[]) => void;
```

Comment:
nit: It seems a little confusing that we refer to screen IDs as `zones` throughout these components (e.g. `routeZones`). The thing that we usually call a "zone" is just the zone identifier within the context of a single SCU, like `m` for Mezzanine. Something like `screenIds` or `signIds` might be clearer / more aligned with the terminology used elsewhere?

Reply:
I agree, and I actually did make that change over in my Select Zones Page branch. I will go ahead and also make it here so that way we merge in what we want when this is approved.